### DISCUSSION TITLE
Different colors in icon and text

### DISCUSSION THREAD

**Discussion by PhosCity** [Alexays/Waybar#2794] [Q&A] [answered]
Just getting started in waybar and I don't see the way to make the icon and the text of same module have different color in waybar. All the examples in the wiki has both icon and text same color too so I was wondering if it is not possible. Or I may just have missed it in the wiki.

**Comment by PhosCity** [comment, marked as answer]
Nevermind, found a hacky solution. Thanks for a great bar app.

![Screenshot_2024-01-05-09-51-26_001_0](https://github.com/Alexays/Waybar/assets/65547311/0745683a-31b4-47b7-acbc-ba14f189dad7)

**Comment by victorballester7** [comment]
Hi!

I was wondering how did you do it?

Thanks!

**Reply by pafael** [reply]
You can use [pango markups](https://docs.gtk.org/Pango/pango_markup.html#pango-markup) in the module format.

For example:
```json
"wireplumber": { 
    "format": "<span foreground=\"blue\">{icon}</span>\n{volume}%"
}
